baybe/searchspace/discrete.py

```diff
@@ -240,10 +245,19 @@ def from_dataframe(
         """
 
         def discrete_parameter_factory(
-            name: str, values: Collection[Any]
+            name: str, values: npt.NDArray
         ) -> DiscreteParameter:
             """Try to create a numerical parameter or use a categorical fallback."""
             try:
+                if pd.api.types.is_bool_dtype(values):
+                    # Due to the difference between bool and np.bool and pandas'
```

Comment:
Just for me to understand, what would the "naive" check even look like and what could go wrong?

Reply:
OK, got it. Thx for the explanation 👍🏼